dev-lang/ghc/ghc-9.4.2.ebuild

```diff
@@ -116,6 +125,7 @@ BDEPEND="
 		>=dev-libs/libxslt-1.1.2 )
 	!ghcbootstrap? ( ${PREBUILT_BINARY_DEPENDS} )
 	test? ( ${PYTHON_DEPS} )
+	${hadrian_deps}
```

Comment:
This should probably go under `DEPEND` instead.

See: https://devmanual.gentoo.org/general-concepts/dependencies/#build-dependencies